modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/AbstractJavaCodegen.java

```diff
@@ -289,8 +290,7 @@ public void processOpts() {
 
         if (additionalProperties.containsKey(ADDITIONAL_MODEL_TYPE_ANNOTATIONS)) {
             String additionalAnnotationsList = additionalProperties.get(ADDITIONAL_MODEL_TYPE_ANNOTATIONS).toString();
-
-            this.setAdditionalModelTypeAnnotations(Arrays.asList(additionalAnnotationsList.split(";")));
+            this.setAdditionalModelTypeAnnotations(Arrays.asList(additionalAnnotationsList.split("\\s*(;|\\r?\\n)\\s*")));
```

Comment:
0 tests for this. please add tests for `;` and the new splitting characters too (including `\r\n` and `\n`

Reply:
I tried in AbstractJavaCodegenTest.java but without getter for additionalModelTypeAnnotations, I found irrelevant to do
```
@Test
public void testAdditionalModelTypeAnnotations() throws Exception {
	OpenAPI openAPI = TestUtils.createOpenAPI();

        final AbstractJavaCodegen codegen = new P_AbstractJavaCodegen();
        codegen.additionalProperties().put(AbstractJavaCodegen.ADDITIONAL_MODEL_TYPE_ANNOTATIONS, "@Foo;@Bar");
        codegen.processOpts();
        codegen.preprocessOpenAPI(openAPI);

        Assert.assertEquals(codegen.additionalProperties().get(AbstractJavaCodegen.ADDITIONAL_MODEL_TYPE_ANNOTATIONS), "@Foo;@Bar");
}
```
Any idea ?

Reply:
Well, I add a getter for additionalModelTypeAnnotations to test the list easily